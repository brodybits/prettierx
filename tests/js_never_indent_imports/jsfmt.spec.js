run_spec(__dirname, ["babel", "babel-flow", "flow", "typescript"], {
  neverIndentImports: true
});
run_spec(__dirname, ["babel", "babel-flow", "flow", "typescript"], {
  neverIndentImports: false
});
