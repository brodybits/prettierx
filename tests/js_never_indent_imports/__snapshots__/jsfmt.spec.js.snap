// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`long_import.js 1`] = `
====================================options=====================================
neverIndentImports: true
parsers: ["flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
import abc, { Lorem, ipsum, dolor, sit, amet, consectetur, adipiscing, elit, sed, eiusmod, tempor, incididunt, ut, labore, et, magna } from "some-module";
import { Lorem, ipsum, dolor, sit, amet, consectetur, adipiscing, elit, sed, eiusmod, tempor, incididunt, ut, labore, et, magna } from "some-module";

=====================================output=====================================
import abc, { Lorem, ipsum, dolor, sit, amet, consectetur, adipiscing, elit, sed, eiusmod, tempor, incididunt, ut, labore, et, magna } from "some-module";
import { Lorem, ipsum, dolor, sit, amet, consectetur, adipiscing, elit, sed, eiusmod, tempor, incididunt, ut, labore, et, magna } from "some-module";

================================================================================
`;

exports[`long_import.js 2`] = `
====================================options=====================================
neverIndentImports: false
parsers: ["flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
import abc, { Lorem, ipsum, dolor, sit, amet, consectetur, adipiscing, elit, sed, eiusmod, tempor, incididunt, ut, labore, et, magna } from "some-module";
import { Lorem, ipsum, dolor, sit, amet, consectetur, adipiscing, elit, sed, eiusmod, tempor, incididunt, ut, labore, et, magna } from "some-module";

=====================================output=====================================
import abc, {
  Lorem,
  ipsum,
  dolor,
  sit,
  amet,
  consectetur,
  adipiscing,
  elit,
  sed,
  eiusmod,
  tempor,
  incididunt,
  ut,
  labore,
  et,
  magna
} from "some-module";
import {
  Lorem,
  ipsum,
  dolor,
  sit,
  amet,
  consectetur,
  adipiscing,
  elit,
  sed,
  eiusmod,
  tempor,
  incididunt,
  ut,
  labore,
  et,
  magna
} from "some-module";

================================================================================
`;
